deploy/playbooks/roles/common/tasks/letsencrypt.yml

```diff
@@ -11,17 +11,28 @@
     update_cache: yes
   become: true
 
-- name: add certbot PPA
-  apt_repository:
-    repo: "ppa:certbot/certbot"
+- name: install snapd
   become: true
-
-- name: install certbot
   apt:
-    name: python-certbot-nginx
-    state: latest
-    update_cache: yes
+    name: snapd
+    state: present
+
+- name: refresh snapd version
   become: true
+  shell:
+    cmd: snap install core; snap refresh core
+
+- name: install certbot snap
```

Comment:
Is this version required for some reason?  I've just been using the standard `certbot` package provided in Ubuntu repos.

Reply:
I've changed the code to just install certbot, but need to test that path (I've been using private self-signed SSL)